src/components/user-tile/user-tile.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactNode, useState } from 'react';
+import { Button } from '@/components/button/button';
+import { ProfilePicture } from '@/components/profile-picture/profile-picture';
+import { PublicUser } from '@/models/public-user';
+import './user-tile.scss';
+
+export interface UserTileProps {
+  user: PublicUser;
+  onClick?: () => void;
+}
+
+export const UserTile = ({ user, onClick }: UserTileProps) => {
+  const userName = user.username;
+  const numDecks = user.publicDecks.length;
+  return (
+    <Button className="user-tile" onClick={() => onClick?.()} variant="invisible">
+      <ProfilePicture username={user.username} showGlow={false} className="user-tile-profile-pic" />
```

Comment:
Yeah, that makes sense.  Should a user have access to the emails of all other users? 

Would be better of sending over a URL? I think that would also make adding a profile picture easier if we decide to implement that.

Reply:
we should probably have a discussion on that since we can get a user's public data via their username OR email
if not, we could also have the backend hash their email and send it instead

because the PFP URL is really just a static endpoint with the only changing data the md5-hashed email